pkg/administration/events/hints.go

```diff
@@ -15,19 +15,30 @@ var (
 	// In the future, we may extend this, and possibly also ensure hints are loaded externally (similar to
 	// vulnerability definitions).
 	hints = map[string]map[string]map[string]string{
+		authenticationDomain: {
+			adminResources.APIToken: {
+				"": `An API token is about to expire. See the details on the expiration time within the event message.
+It is not possible to re-create the token, instead you have to do the following:
+- Delete the expiring API token.
+- Create a new API token (you may choose the same name).
+
+Afterwards, you may use the newly created API token.
```

Comment:
suggestion:

You can then use the newly-created API token.

(we don't use afterwards or may)